c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -1205,7 +1205,7 @@ class DeviceCachingAllocator {
   // outstanding cuda events
   ska::flat_hash_map<
       cuda::CUDAStream,
-      std::deque<std::pair<EventPool::Event, Block*>>>
```

Comment:
f here is probably a typo

Reply:
Ahh! - Thanks. Fixed and merged latest master